index.js

```diff
@@ -189,6 +206,9 @@ function launchVideo(opts) {
   opts.getUrlFn(id, function(err, streamUrl) {
     if (!err) updatePanel({src: streamUrl});
   });
+
+  // todo: see if we can just call this when initializing the panel
+  makePanelTransparent(panel);
```

Comment:
@meandavejustice Yeah, it's possible to just call this on startup, I'll file a bug